legend-engine-pure-code-compiled-core/src/main/resources/core/legend/service/metamodel.pure

```diff
@@ -223,17 +224,6 @@ Diagram meta::legend::service::metamodel::ServiceDiagram(width=0.0, height=0.0)
         color=#FFFFCC,
```

Comment:
Any reason we deleted contents from Diagram?

Reply:
https://github.com/finos/legend-engine/pull/1889